src/api/resolvedTimeline.ts

```diff
@@ -0,0 +1,135 @@
+import { Content, TimelineObject } from './timeline'
+import { InstanceId, Reference, Time } from './types'
+
+/**
+ * The ResolvedTimeline contains all objects on the timeline, resolved.
+ * All references and all conflicts have been resolved.
+ * The absolute times of all objects can be found in objects[id].resolved.instances.
+ *
+ * To retrieve a state at a certain time, use getResolvedState(resolvedTimeline).
+ *
+ * Note: If `limitTime` was specified in the ResolveOptions,
+ * the ResolvedTimeline is only valid up until that time and
+ * needs to be re-resolved afterwards.
+ */
+export interface ResolvedTimeline<TContent extends Content = Content> {
+	/** The options used to resolve the timeline */
+	// options: ResolveOptions
+	/** Map of all objects on timeline */
+	objects: ResolvedTimelineObjects<TContent>
+	/** Map of all classes on timeline, maps className to object ids */
+	classes: { [className: string]: Array<string> }
+	/** Map of the object ids, per layer */
+	layers: { [layer: string]: Array<string> }
+
+	// states: AllStates
+	nextEvents: Array<NextEvent>
+
+	statistics: {
+		/** Number of objects that were unable to resolve */
+		unresolvedCount: number
+		/** Number of objects that were resolved */
+		resolvedCount: number
```

Comment:
Yeah I actually think that we should remove this counter altogether. It's a remnant from an old version where (iirc) objects that didn't resolve to any instances where counted as "unresolved", however it was badly documented and caused confusion (me included).
In version 8 it functioned as an "all objects" counter iirc, which isn't that useful I think.

Reply:
I've removed `unresolvedCount` now (it didn't make sense anymore since it would always be 0) and renamed `resolvedCount` to `totalCount` and changed the description.